packages/input-manager/src/devices/feedbackStore.ts

```diff
@@ -0,0 +1,35 @@
+import { SomeFeedback } from '../feedback/feedback'
+
+export class FeedbackStore<T extends SomeFeedback> {
```

Comment:
Question: You're not really using this class as a generic anywhere, perhaps just use `SomeFeedback` directly?

Reply:
Well, I'm not using it right now, but the plan was that the FeedbackStore could be used for objects enhancing the SomeFeedback interface, that's why it's generic.